.travis.yml

```
@@ -26,13 +26,15 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true)
```

Please, change this to `travis-cargo install rustfmt` (avoid using `|| true`, since, as I said, it can lead to errors being suppressed.

---

did that because of
```bash
error: binary `cargo-fmt` already exists in destination as part of `rustfmt v0.6.2`
binary `rustfmt` already exists in destination as part of `rustfmt v0.6.2`
Add --force to overwrite
```

---

and also forcing rustfmt install on every build will increase build time.

---

OK, yes, I understand :)